### PR TITLE
docs: Add Juno AWS Node Runner

### PR DESCRIPTION
**Pull request by odednaor**
### Description of the Changes

Add [Juno AWS Node Runner](https://aws-samples.github.io/aws-blockchain-node-runners/docs/Blueprints/Starknet) to the docs.

### PR Preview URL

[Full nodes and API services](https://starknet-io.github.io/starknet-docs/pr-1279/documentation/tools/api-services/)

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1279)

### CODE REVIEW

File: components/Starknet/modules/tools/pages/api-services.adoc [hunk 1]
```diff
@@ -50,9 +50,11 @@ https://docs.data.voyager.online/spec[RPC spec versions]
...
 [%header,cols="2,2,1"]
 |===
-| Provider name | Description | More information
-|Deoxys|A Starknet full-node written in Rust and powered by Substrate by Kasar |link:https://github.com/kasarlabs/deoxys[github.com/kasarlabs/deoxys]
-|Juno|A Starknet full-node written in go-lang by Nethermind |link:https://github.com/NethermindEth/juno[github.com/NethermindEth/juno]
-|Papyrus|A Starknet full-node written in Rust by StarkWare | link:https://github.com/starkware-libs/papyrus[github.com/starkware-libs/papyrus]
-|Pathfinder|A Starknet full-node written in Rust by Equilibrium |link:https://github.com/eqlabs/pathfinder[github.com/eqlabs/pathfinder]
+| Provider name | Description | More information 
+|Deoxys|A Starknet full-node written in Rust and powered by Substrate by Kasar |link:https://github.com/kasarlabs/deoxys[github.com/kasarlabs/deoxys] 
+|Juno|A Starknet full-node written in go-lang by Nethermind |link:https://github.com/NethermindEth/juno[github.com/NethermindEth/juno]
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/api-services.adoc:55]:
```suggestion
|Juno|A Starknet full-node written in go-lang by Nethermind.

You can use the link:https://aws-samples.github.io/aws-blockchain-node-runners/docs/Blueprints/Starknet[Juno Node runner on AWS].
|link:https://github.com/NethermindEth/juno[github.com/NethermindEth/juno] 
```

File: components/Starknet/modules/tools/pages/api-services.adoc [hunk 2]
```diff
@@ -50,9 +50,11 @@ https://docs.data.voyager.online/spec[RPC spec versions]
...
-| Provider name | Description | More information
-|Deoxys|A Starknet full-node written in Rust and powered by Substrate by Kasar |link:https://github.com/kasarlabs/deoxys[github.com/kasarlabs/deoxys]
-|Juno|A Starknet full-node written in go-lang by Nethermind |link:https://github.com/NethermindEth/juno[github.com/NethermindEth/juno]
-|Papyrus|A Starknet full-node written in Rust by StarkWare | link:https://github.com/starkware-libs/papyrus[github.com/starkware-libs/papyrus]
-|Pathfinder|A Starknet full-node written in Rust by Equilibrium |link:https://github.com/eqlabs/pathfinder[github.com/eqlabs/pathfinder]
+| Provider name | Description | More information 
+|Deoxys|A Starknet full-node written in Rust and powered by Substrate by Kasar |link:https://github.com/kasarlabs/deoxys[github.com/kasarlabs/deoxys] 
+|Juno|A Starknet full-node written in go-lang by Nethermind |link:https://github.com/NethermindEth/juno[github.com/NethermindEth/juno] 
+
+link:https://aws-samples.github.io/aws-blockchain-node-runners/docs/Blueprints/Starknet[Juno AWS Node Runner]
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/api-services.adoc:57]:
```suggestion
```

**Review by stoobie — COMMENTED** [review]
@odednaor I think it makes more sense to put this in the middle column. WDYT?

**Review by odednaor — COMMENTED** [review]
lgtm

**Review by stoobie — APPROVED** [review]

**Pull request merged**